File: java program/inheritance/rectangle.java
class rectangle
{
    int area;
    rectangle(int len, int brea)
    {
         area=len*brea;
    }
    void ayush()
    {
         System.out.println("area: "+ area);

    }
    
    public static void main(String[] args)
    {
        rectangle rect=new rectangle(3, 5);
        rect.ayush();
        

        
    }
}

File: c++ programe/function/rectangle.cpp
#include<iostream>
using namespace std;
class rectangle
{
    public:                                 
    int length, breadth;
    rectangle(int l, int b)
    {
        length=l;
        breadth=b;
    }

    void getarea()
    {
      int area = length * breadth;
      cout<<"Area: "<<area;
    }
};

int main()
{
    rectangle rect(7, 7);
    rect.getarea();
    return 0;
}